src/api_recievers/influxqueries.js

```diff
@@ -28,7 +93,7 @@ const appData = async (source, appName, currentSession) =>{
 
   if(write === 'true' && (prevAppName === undefined || prevAppName !== appName)) {
     log.debug("Writing to db");
-    let writeClient = client.getWriteApi(org, bucket, 's');
+    let writeClient = influxClient.getWriteApi(org, influxBuckets.apps, 's');
```

Comment:
additionally, if you pass influxBuckets.apps into getWriteApi, will we have to modify this later to write to the other buckets for tasks